scheduler/src/cook/mesos/util.clj

```diff
@@ -28,6 +28,49 @@
           db)
        (map first)))
 
+(defn job-ent->container
+  "Take a job entity and return its container"
+  [db job job-ent]
+  (clojure.pprint/pprint "=job-ent->container=")
+  (if (contains? job-ent :job/container)
+    (let [ceid (:db/id (:job/container job-ent))
+          cmap (d/pull db "[*]" ceid)
+          rm-dbids (fn rm-dbids [m]
+                     (cond
+                       (map? m)
+                       (let [sm (filter (fn [p]
+                                          (not(= :db/id (first p)))) m)]
+                         (into {} (map (fn [p]
+                                         [(first p) (rm-dbids (second p))])
+                                       sm)))
+                       (vector? m)
+                       (mapv rm-dbids m)
+                       :else
+                       m))
+          fixup-keywords (fn fixup-keywords [m]
+                           (cond
+                             (map? m)
+                             (into {} (map (fn [p]
+                                             (let [k (first p)
+                                                   v (second p)]
+                                               [ (if (keyword? k)
+                                                   (keyword(name k)) k)
```

Comment:
I was thinking that maintaining the datomic/name -> name mapping would allow the code continue to work even as the schema gets extended under new versions of mesos.

Would you consider documentation as a solution or do you want  a keyword -> keyword table ?